src/packages/excalidraw/README_NEXT.md

```diff
@@ -904,3 +906,7 @@ getFreeDrawSvgPath(element: <a href="https://github.com/excalidraw/excalidraw/bl
 </pre>
 
 This function returns the free draw svg path for the element.
+
+## Need help?
+
+Check out the existing [QnA's](https://github.com/excalidraw/excalidraw/discussions?discussions_q=label%3Apackage%3Aexcalidraw). If you have any queries or need any help, ask us [here](https://github.com/excalidraw/excalidraw/discussions?discussions_q=label%3Apackage%3Aexcalidraw).
```

Comment:
```suggestion
Check out the existing [Q&A](https://github.com/excalidraw/excalidraw/discussions?discussions_q=label%3Apackage%3Aexcalidraw). If you have any queries or need help, ask us [here](https://github.com/excalidraw/excalidraw/discussions?discussions_q=label%3Apackage%3Aexcalidraw).
```

I don't think QnA abbreviation is used in this context. Also, mind the apostrophes :)